### PR TITLE
Fix input checkbox's state

### PR DESCRIPTION
**Pull request by gfx**
`HTML2React(<input type="checkbox" checked readonly disbled/>)` does not work as expected.

This PR fills correct values to boolean attributes.

### CODE REVIEW

File: src/get-props-from-attributes.js [hunk 1]
```diff
@@ -64,5 +64,13 @@ function getPropValue (propName, attrValue) {
     return getStylePropValue(attrValue)
   }
 
+  switch (lowerPropName) {
```
Comment (gfx) [inline src/get-props-from-attributes.js:67]: This is is the core of this PR. All the other changes are for testing.

File: test/html2react.test.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import HTML2React from '../src';
+
+const dataset = [
+  {
+    input: "<p>Hello!</p>",
+    output: '<p>Hello!</p>',
+  },
+  {
+    input: "<input type='checkbox' checked/>",
+    output: '<input type="checkbox" checked=""/>',
```
Comment (gfx) [inline test/html2react.test.jsx:15]: I prefer `checked` without `=""`, but it's what ReactDOMServer does and it just works.

**Comment by Deschtex** [comment]
Hey, @gfx. Thanks for contributing! I had overlooked these particular attributes. Also, +1 for adding tests (I've been meaning to 😁).

**Review by Deschtex — APPROVED** [review]
LGTM!

**Comment by gfx** [comment]
Thanks for the quick reply! 😄

**Pull request merged**